dremio_client/query.py

```diff
@@ -39,6 +39,7 @@
 def query(
     token,
     base_url,
+    rest_url,
```

Comment:
`rest_url` takes the place of `base_url` so we can remove `base_url` here. But they are the same value so I don't understand the purpose?

Reply:
@3runkenzie?

Reply:
Yes. I wanted to keep the parameters you had there initially (hence `base_url` does not get used) to incorporate a concept of having 2 types of urls.
<pre>
self._rest_url = (
            ("https" if config["ssl"].get(bool) else "http")
            + "://"
            + self._hostname
            + (":{}".format(<b>self._rest_port</b>) if port else "")
        )
</pre>



<pre>
self._base_url = (
            ("https" if config["ssl"].get(bool) else "http")
            + "://"
            + self._hostname
            + (":{}".format(<b>port</b>) if port else "")
        )
</pre>

since the ports are different for ODBC vs REST.